src/west/manifest-schema.yml

```diff
@@ -95,6 +95,10 @@ mapping:
           path:
             required: false
             type: str
+          # Submodules to checkout within the project
+          submodules:
+            required: false
+            type: any
```

Comment:
It it possible to use:
```
type: seq
sequence:
  - type: str
```
?

Reply:
I guess yes. I did it that way, because I wasn't sure what will be the final internal structure of submodules.

Reply:
Sorry I was wrong, it can't be str, as submodules contains internal components, so it's not string. Actually I think I did it wrong (it worked, but could be better), so now I changed submodules type to sequence and I also defined types for the name and path values. I have a hope that this is better approach, as 'projects' structure looks right this way.